src/cli/assert_node_engine_version_spec.ts

```diff
@@ -1,18 +1,37 @@
-import assert from 'assert'
-import { assertNodeEngineVersion } from './assert_node_engine_version'
+import { expect } from 'chai'
+import * as sinon from 'sinon'
+import { validateNodeEngineVersion } from './assert_node_engine_version'
 
-describe(assertNodeEngineVersion.name, () => {
-  it('fails when the version is lower than specified in package.json', () => {
-    assert.throws(() =>
-      assertNodeEngineVersion('v11.0.0', () => ({
-        engines: {
-          node: '>=12',
-        },
-      }))
+describe(validateNodeEngineVersion.name, () => {
+  it('returns an error message when the version is lower than specified in package.json', () => {
+    // Arrange
+    const errorSpy = sinon.spy()
+
+    // Act
+    validateNodeEngineVersion('v11.0.0', errorSpy, () => ({
+      engines: {
+        node: '>=12',
+      },
+    }))
+
+    // Assert
+    expect(errorSpy).to.have.been.calledOnceWith(
+      'Cucumber can only run on Node.js versions >=12. This Node.js version is v11.0.0'
     )
   })
 
-  it('passes when the version is greater than specified in package.json', () => {
-    assertNodeEngineVersion('v17.0.0')
+  it('returns null when the version is greater than specified in package.json', () => {
```

Comment:
I just realised we should have changed these example descriptions too: we're not returning anything anymore.